components/frontend/src/report/IssueTracker.js

```diff
@@ -130,10 +154,18 @@ export function IssueTracker({ report, reload }) {
                         allowAdditions
                         id="tracker-issue-labels"
                         requiredPermissions={[EDIT_REPORT_PERMISSION]}
-                        label="Labels for new issues"
+                        label={<label>Labels for new issues <Popup on={['hover', 'focus']} content={"Spaces in labels are allowed here, but they will be replaced by underscores in Jira"} trigger={<Icon tabIndex="0" name="help circle" />} /></label>}
```

Comment:
What happens when the user inputs a label with spaces and the same but with an underscore?

Reply:
Good question...